fixtures/next-pages-router/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@fixtures/next-pages-router",
+  "description": "Next pages router fixtures",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "clean-build": "pnpm clean && next build",
+    "clean": "pnpm clean:dev && pnpm clean:prod",
+    "clean:dev": "rm -rf .next",
+    "clean:prod": "rm -rf dist"
+  },
+  "dependencies": {
+    "@fixtures/features": "*",
+    "@fixtures/recipes": "*",
+    "@fixtures/sprinkles": "*",
+    "@vanilla-extract/css": "*",
+    "@vanilla-extract/recipes": "*",
+    "@vanilla-extract/sprinkles": "*",
+    "next": "12.3.4",
+    "react": "npm:react@^17.0.2",
+    "react-dom": "npm:react-dom@^17.0.2"
+  },
+  "devDependencies": {
+    "@types/react": "npm:@types/react@^17",
```

Comment:
The rest of the repo is on react 18 as of https://github.com/vanilla-extract-css/vanilla-extract/pull/1326. Is there a reason we need react 17 specifically, or was this leftover from splitting off from your original PR? Does next v12 not support react 18?

Reply:
yeah this must have been leftover from before everything was moved to react 18, will change